### PR TITLE
api/orders

### PR DESCRIPTION
**Pull request by Scarlette-O**
check the code. let me know if i need to make changes.

### CODE REVIEW

File: api/orders.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+router.use(bodyParser.json());
+
+const {
+    createOrder,
+    updateOrder,
+    deleteOrder
+} = require('../db/models/orders');
+
+const {
```
Comment (cmceachin5888) [inline api/orders.js:14]: seems like duplicate requires no?

**Review by cmceachin5888 — CHANGES_REQUESTED** [review]

File: api/orders.js (hunk 1 above)
Comment (Scarlette-O) [reply api/orders.js:14]: I dont know what i was thinking but yes it is a dup. Ill make that change.

**Review by Vimaser — APPROVED** [review]
Looks good!

**Review by cmceachin5888 — APPROVED** [review]

**Comment by Scarlette-O** [comment]
Reviewed and Ok'd by two other members.

**Pull request merged**